optuna/pruners/hyperband.py

```diff
@@ -1,14 +1,14 @@
-import optuna
+import numpy
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+import warnings
```

Comment:
`numpy` is a third-party library. Please reorder these import lines and add a blank line as follows:
```suggestion
from typing import List  # NOQA
from typing import Optional  # NOQA
import warnings

import numpy
```